engines/glk/scott/scott.cpp

```diff
@@ -403,9 +403,9 @@ void Scott::look(void) {
 	if ((_bitFlags & (1 << DARKBIT)) && _items[LIGHT_SOURCE]._location != CARRIED
 			&& _items[LIGHT_SOURCE]._location != MY_LOC) {
 		if (_options & YOUARE)
-			display(_topWindow, _("You can't see. It is too dark!\n"));
+			display(_topWindow, _("You can't see. It is too dark!\n").encode().c_str());
 		else
-			display(_topWindow, _("I can't see. It is too dark!\n"));
+			display(_topWindow, _("I can't see. It is too dark!\n").encode().c_str());
```

Comment:
this

Reply:
I don't think using UTF-8 will work correctly. To make this work I think we would need to keep U32Strings here and add a display() function that takes `const uint32 *` instead of `const char *` and calls `glk_put_string_stream_uni()` instead of `glk_put_string_stream()`.